pkg/languagedetection/detector.go

```diff
@@ -39,20 +45,21 @@ var exactMatches = map[string]languageFromCLI{
 	"node": {name: languagemodels.Node},
 
 	"dotnet": {name: languagemodels.Dotnet},
-}
 
-func languageNameFromCommandLine(cmdline []string) languagemodels.LanguageName {
-	exe := getExe(cmdline)
+	"ruby":  {name: languagemodels.Ruby},
+	"rubyw": {name: languagemodels.Ruby},
+}
 
+func languageNameFromCommand(command string) languagemodels.LanguageName {
```

Comment:
I wonder if it's worth passing in the values from `/proc/pid/comm` and `/proc/pid/commandline`, doesn't have to be in this PR though

Reply:
I opted to leave this function as is because we execute the same logic for both the `comm` value and the first arg from `cmdline`. If you think it makes sense, I can encapsulate the two calls below into its own function that takes in a `*procutil.Process`.

```
		exe := getExe(proc.Cmdline)
		languageName := languageNameFromCommand(exe)
		if languageName == languagemodels.Unknown {
			languageName = languageNameFromCommand(proc.Comm)
		}
```

Reply:
I think it's okay to leave it as it is then